client/slack.go

```diff
@@ -147,6 +147,20 @@ func (sl *slackClient) handleMessageEvents() {
 				continue
 			}
 			sl.incomingMessages <- toFlyteMessageEvent(v, u)
+
+		case *slack.ReactionAddedEvent:
+			u, err := sl.client.GetUserInfo(v.User)
+			log.Debug().Msgf("received reaction event for type = %v", v)
```

Comment:
FYI, you can use structured logging e.g. 

`log.Debug().Str("type", v).Msg("received reaction event")`

The only thing is v is not a type and message itself is misleading. What you are logging is event data.

Reply:
Move it above the `GetUserInfo` call since you are logging that you have received a reaction event and it should be your first call before you will start processing this event

Reply:
after looking into other code i believe what you've meant by type is `v.Type` which will be available after casting v to correct type

Reply:
Actually, yes, I would change the trace, I want to trace the complete payload received from the slack client in case of error scenario's.